### PR TITLE
Add pollen sensors to Ambee

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

Add information about the pollen sensors to the Ambee documentation.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/51702
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/ambee.markdown [hunk 1]
```diff
@@ -27,8 +27,10 @@ themselves.
 
 To use the Ambee integration, you will need to obtain an API key from Ambee
 
-Ambee provides a free API keys, that are limited to 100 requests a day. If
-you are in need for more, paid options are available as well.
+Ambee provides a free API keys, that are limited to 100 requests a day. This
```
Comment (MartinHjelmare) [inline source/_integrations/ambee.markdown]:
```suggestion
Ambee provides free API keys, that are limited to 100 requests a day. This
```

File: source/_integrations/ambee.markdown [hunk 2]
```diff
@@ -53,3 +55,36 @@ various health risks. The following sensors from Ambee are available on this:
...
+risks.
+
+Generic pollen count sensors (in pollen/m3)
+and as risk level (low, moderate, high, very high):
+
+- Grass
+- Tree
+- Weed
+
+Additionally, sensors are provided for specific pollen from specific grasses,
```
Comment (MartinHjelmare) [inline source/_integrations/ambee.markdown]:
```suggestion
Additionally, sensors for specific pollen from specific grasses,
```

File: source/_integrations/ambee.markdown [hunk 3]
```diff
@@ -53,3 +55,36 @@ various health risks. The following sensors from Ambee are available on this:
...
+trees or weeds, in pollen/m3, are provided. These are disabled by default,
+enable those entities in the user interface if you like to use these:
```
Comment (MartinHjelmare) [inline source/_integrations/ambee.markdown]:
```suggestion
trees or weeds, in pollen/m3, are provided. These are disabled by default.
Enable those entities in the user interface if you like to use these:
```

**Review by MartinHjelmare — APPROVED** [review]

**Review by klaasnicolaas — APPROVED** [review]
✅

**Pull request merged**